semgrep/semgrep/meta.py

```diff
@@ -475,6 +475,254 @@ def to_dict(self) -> Dict[str, Any]:
         }
 
 
+@dataclass
+class CircleCIMeta(GitMeta):
+    """Gather metadata from Circle CI."""
+
+    environment: str = field(default="circle-ci", init=False)
```

Comment:
```suggestion
    environment: str = field(default="circleci", init=False)
```

cuz they don't use a dash: https://github.com/circleci